file/src/main/scala/akka/stream/alpakka/file/javadsl/Archive.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.file.javadsl
+
+import akka.NotUsed
+import akka.stream.alpakka.file.{scaladsl, ArchiveMetadata}
+import akka.stream.javadsl.Flow
+import akka.util.ByteString
+import akka.japi.Pair
+import akka.stream.javadsl.Source
+
+/**
+ * Java API.
+ */
+object Archive {
+
+  /**
+   * Flow for compressing multiple files into one ZIP file.
+   */
+  def zipFlow(): Flow[Pair[ArchiveMetadata, Source[ByteString, NotUsed]], ByteString, NotUsed] =
+    Flow
+      .create[Pair[ArchiveMetadata, Source[ByteString, NotUsed]]]()
+      .map(pair => (pair.first, pair.second.asScala))
```

Comment:
This line fails to compile in Scala 2.11. Please help the compiler by wrapping the function with `func` as it is done in other connectors: https://github.com/akka/alpakka/blob/261645f14b6a99e36c8f61c7887f878c7883b563/s3/src/main/scala/akka/stream/alpakka/s3/javadsl/S3.scala#L392

Reply:
Thanks! I was struggling a little with it